app/assets/v2/js/pages/bounty_details.js

```diff
@@ -777,20 +775,19 @@ var do_actions = function(result) {
   // Find interest information
   const is_interested = is_current_user_interested(result);
 
+  const has_fulfilled = result['fulfillments'].filter(fulfiller => fulfiller.fulfiller_github_username === document.contxt['github_handle']).length > 0;
+
```

Comment:
not directly related to this task, but whats happens in the app if the user changes his username in github?  it also changes in gitcoin to the urls (profile/username)  and username in every place?

Reply:
Oooo that would mess up everything at our end 🤔 
cc @mbeacom 

we would have to build that out to update all references 🤔